moduleframework/helpers/openshift_helper.py

```diff
@@ -54,8 +58,52 @@ def __init__(self):
         # application name is taken from docker.io/modularitycontainer/memcached
         self.app_name = self.container_name.split('/')[-1]
         self.app_ip = None
+        self.project_name = "%s-%s" % (''.join(random.choice(string.lowercase) for _ in range(3)), self.app_name)
```

Comment:
I would actually do it the other way around: `{project_name}-{random_str}` -- you'll have nice grouping once sorting alphabetically